ush/parsing_namelists_FV3.sh

```diff
@@ -59,6 +59,11 @@ else
   ${NCP} "${FIELD_TABLE}" field_table
 fi
 
+# set resolution dependent variables
+if [[ ${do_gsl_drag_ls_bl:-.true.} == ".true." ]]; then
+   export cdmbgwd=${cdmbgwd_gsl}
+fi
```

Comment:
Is this a resolution dependent variable?  
Why can't this be elevated to `config.fcst` under the if-block of `if ((gwd_opt = 2))`?

Reply:
@aerorahul Yes, 'cdmbgwd' is a resolution dependent variable.  It also depends on the orographic gravity wave drag (OGWD) scheme being called.  Both the 'old' GFSv16 and 'new' GSL OGWD schemes can each be called with 'gwd_opt=2'.  What switches between them is the namelist variable 'do_gsl_drag_ls_bl'.  I originally tried elevating to config.fcst, but this 'do_gsl_drag_ls_bl' is not passed into the script.  Modifying 'parsing_namelists_FV3.sh' seemed the best way to go.

Reply:
I see in `config.fcst` when `gwd_opt == 2`, `do_gsl_drag_ls_bl` is set to `.true.`
Why can't `cdmbgwd=${cdmbgwd_gsl}` in that block?
Is `do_gsl_drag_ls_bl` going to be `.true.` when `gwd_opt != 2`?  

Yes, I understand mountain blocking and gravity wave drag coefficients are resolution dependent.  However, that choice is made in `config.ufs`, and not here. 

I don't see a technical reason why we cannot do this in `config.fcst`

Reply:
No, 'do_gsl_drag_ls_bl' will not be .true. when gwd_opt != 2.  I can try your suggestion this evening or tomorrow when Hera is back up from maintenance.  HOWEVER, I think I tried something similar to what you suggested, but in config.fcst, the variable 'cdmbgwd_gsl', which is set in 'config.ufs', wasn't available and the script would exit with an error.

Reply:
The current method of parsing 'cdmbgwd' into 'input.nml' is directly from 'config.ufs' to 'ush/parsing_namelists_FV3.sh' ('config.fcst' is bypassed).  The proposed method for 'cdmbgwd = cdmbgwd_gsl' follows this same approach, we only need the 'if [[ ${do_gsl_drag_ls_bl:-.true.} == ".true." ]]; then' statement in 'parsing_namelists' to achieve the same end when using the GSL drag parameterizations.  As mentioned earlier, the 'cdmbgwd_gsl' variable wasn't accessible in 'config.fcst'.

Reply:
I don't understand this. 
`config.fcst` sources `config.ufs` on [line 21](https://github.com/NOAA-EMC/global-workflow/blob/5183c43bbbd07285869feacb49d8680afe85929a/parm/config/gfs/config.fcst#L21).
At this point, one should have both `cdmbgwd` and `cdmbgwd_gsl` available.

Reply:
You're right.  I see this.  Once Hera is back up, I'll retry setting 'cdmbgwd=cdmbgwd_gsl' in 'config.fcst'.  Stand by.

Reply:
@aerorahul I tested on Hera and your suggested changes work.  I remember now that I was trying to use 'do_gsl_drag_ls_bl' into 'config.ufs' to set 'cdmbgwd', but that didn't work.  In 'config.fcst' an 'if' statement is still necessary because a user could change the default 'do_gsl_drag_ls_bl' for 'gwd_opt=2' from .true. to .false. and cdmbgwd would need to be changed accordingly.  My modifications to 'config.fcst' assures this.